### PR TITLE
[T5A1][W13-B1] Lim Hui Qi

### PR DESCRIPTION
**Pull request by LuMiN0uSaRc**

### CODE REVIEW

**Comment by LuMiN0uSaRc** [comment]
Use cases 
1. Need to confirm at use case 
2. Print error message 
3. End immediately when there is a tag already

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -74,12 +78,34 @@ Use case ends.
...
   Use case resumes at step 2
 
+#### Use case: Edit tag
+
+
+**MSS**
+
+1. User requests to list persons
+2. AddressBook shows a list of persons
+3. User requests to edit tag of a specific person in the list
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: there should be another round of confirmation between the address book and user.

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -74,12 +78,34 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Edit tag
+
+
+**MSS**
+
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: the activity was updated to rename a tag such that all people with the given tag will have the tag updated. please check the description again

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -74,12 +78,34 @@ Use case ends.
...
+
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Response time of commands should not exceed 3 seconds.
+6. Should store data as a text file in a specified location.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: isn't this a functionality?

**Comment by okkhoy** [comment]
@LuMiN0uSaRc 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**